shared/aux.go

```diff
@@ -45,7 +46,7 @@ func RunCommandOnNode(cmd, ip string) (string, error) {
 	if cmd == "" {
 		return "", ReturnLogError("cmd should not be empty")
 	}
-
+	LogLevel("debug", fmt.Sprintf("EXECUTE: %s on %s", cmd, ip))
```

Comment:
why this here ?

Reply:
to log the commands run - even if there was no error. this helps with debugging and get the flow and figure out which node the command is being run on etc. it is better to add this here instead of adding particular logs outside everytime a run command method is called. this method knows the cmd and where it runs the same automatically.

Reply:
nice. 
do you have a example output to show us here?

Reply:
```
DEBU[2023-11-17T00:22:24Z]/go/src/github.com/rancher/distros-test-framework/shared/aux.go:314 github.com/rancher/distros-test-framework/shared.LogLevel EXECUTE: sudo ls -lt /var/lib/rancher/rke2/server/ | grep tls | awk {'print $9'} | sed -n '2 p' on 3.149.246.214 
DEBU[2023-11-17T00:22:25Z]/go/src/github.com/rancher/distros-test-framework/shared/aux.go:314 github.com/rancher/distros-test-framework/shared.LogLevel StdOut: tls-1700180360 StdErr:  
```